scipy/optimize/__lbfgsb.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __LBFGSB_H
+#define __LBFGSB_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif /* __cplusplus */
+
+#include <math.h>
+
+void
+setulb(const int,const int,double*,double*,double*,int*,double*,double*,
```

Comment:
Can you add documentation onto what each parameter represents, and their required sizes? this info can be adapted from the Fortran.

Reply:
Yes typically I do that in the last commit to have the last shape. This is just to get the compilation mechanism to put in place.